examples/small-study/src/background.js

```diff
@@ -58,14 +57,18 @@ class FeatureToInstrument {
    * - set up expiration alarms
    * - make feature/experience/ui with the particular variation for this user.
    */
-  async enableFeature(studyInfo) {
+  async enableFeature(studyInfo, isFirstRun) {
```

Comment:
I claim that isFirsRun should be a key in study info.

Reply:
I left it out from studyInfo since it has no meaning outside of the onReady event handler. Claim: It will always be false anytime getStudyInfo() is invoked, unless it is invoked within the onReady event handler.

Reply:
I think it's okay to leave it in, because it has a lot of utility.  It is a proxy for the study 'state'.
I also / like / strongly prefer the symmetry that `setup` just returns info.   One schema to check everywhere.

Reply:
Sure, will bake this into studyInfo by hard-coding it to false by default and overriding it to true just for the onReady event parameter.